### DISCUSSION TITLE
Can I run build docker image from cloned frigate repo?

### DISCUSSION THREAD

**Discussion by hassanalikhan87** [blakeblackshear/frigate#14274] [Ask A Question] [answered]
I want to add my custom code on top of frigate but docker-compose.yml file is not building it. Can I get any help?

**Comment by NickM-27** [comment, marked as answer]
You shouldn't be building from docker compose you should build using make

**Reply by hassanalikhan87** [reply]
Thank you so much Nick. It works now in Docker.

Is there any way I can decouple the frontend and backend?

I already have the frontend running separately but cannot get the backend to run.

**Reply by NickM-27** [reply]
Not sure I understand what you mean, it is decoupled. You can run the frontend pointed at whatever frigate instance you want

**Reply by hassanalikhan87** [reply]
I know.
But I want to run backend in a different port.

**Reply by NickM-27** [reply]
So change the port in docker config where it is deployed

**Reply by hassanalikhan87** [reply]
Got it. Thank you so much for your help. Means a lot.
